sdk/src/androidTest/java/com/mapbox/maps/OfflineTest.kt

```diff
@@ -444,15 +438,13 @@ class OfflineTest {
     val latch = CountDownLatch(1)
     var idleEventCount = 0
     var mapLoadingErrorCount = 0
-    val observer = object : Observer() {
-      override fun notify(event: Event) {
-        Logger.e(TAG, "type ${event.type}, data ${event.data.toJson()}")
-        if (event.type == MapEvents.MAP_IDLE) {
-          idleEventCount++
-        }
-        if (event.type == MapEvents.MAP_LOADING_ERROR) {
-          mapLoadingErrorCount++
-        }
+    val observer = Observer { event ->
```

Comment:
duplicate [408-416]

Reply:
I'll not be doing refactoring existent code as part of this PR, this PR is to adopt to latest core only.
We can revisit those nit's later as part of some other cleanup.